app/controllers/ops_controller/diagnostics.rb

```diff
@@ -736,14 +736,25 @@ def diagnostics_set_form_vars
     end
   end
 
-  def database_details
+  def row_data(label, value)
```

Comment:
Hey @GilbertCherrie , can we move these to `ops_helper.rb` (like we did in all other modules).

By doing so, we can change the `haml` extension of `_diagnostics_database_tabe.html.haml` to `erb`